api/src/main/java/uk/gov/hmcts/payment/api/dto/BulkScanPaymentRequestStrategic.java

```diff
@@ -0,0 +1,104 @@
+package uk.gov.hmcts.payment.api.dto;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.experimental.Wither;
+import org.apache.commons.lang.StringUtils;
+import org.joda.time.DateTime;
+import uk.gov.hmcts.payment.api.contract.PaymentAllocationDto;
+import uk.gov.hmcts.payment.api.contract.util.CurrencyCode;
+import uk.gov.hmcts.payment.api.contract.util.Service;
+import uk.gov.hmcts.payment.api.model.PaymentChannel;
+import uk.gov.hmcts.payment.api.model.PaymentStatus;
+import uk.gov.hmcts.payment.api.util.PaymentMethodType;
+
+import javax.validation.constraints.*;
+import java.math.BigDecimal;
+
+import static com.fasterxml.jackson.annotation.JsonInclude.Include.NON_NULL;
+
+@JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class)
+@JsonInclude(NON_NULL)
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder(builderMethodName = "createBulkScanPaymentStrategicWith")
+public class BulkScanPaymentRequestStrategic {
```

Comment:
What is the purpose of duplicating an existing Object[BulkScanPaymentRequest], can we not reuse the existing Object?

Reply:
New pojo is created to separately carry out new changes for new endpoints

1) This will ensure no changes on existing methods and existing classes, thereby reducing the risk of regression failures. 
2) It will ensure no impact on existing endpoint
3) In future when we would be removing old endpoints, it will simplify refactoring due to the separation of POJO and methods

Reply:
Duplication of existing code is simply considered as CPD violation, re-usability should always be entertained.

Reply:
it's not a duplicate of code, and we cant use the same class because existing endpoints request will be changed

Reply:
I agree with Amit, I prefer to create a new DTO than use the existing one

Reply:
If my understanding is correct, the requirement was to make a call to BSP from Payhub and update the DCN Status to Processed when PayBubble is calling PayHub to allocate a BSP Payment, so to achieve that I think we should modify the existing API with this new requirement regulated by a Feature Flag.

1. But I can't agree to this point that for an amendment to an existing feature we should create redundant Models[Inherited model should have been used] & APIs which would become a candidate for clean-up(additional effort), also how should we justify our effort to program for this clean up.
2. By propagating the Paybubble HttpHeaders to BSP through PayHub, we are sending PayBubble S2SToken to BSP in a request made by PayHub to BSP, we should send PayHub S2S Token to BSP to get the request Authenticated. I believe this is strongly against security.